manifests/fetch.pp

```diff
@@ -20,18 +20,26 @@
     false => '--silent --show-error'
   }
 
+  if $sha {
```

Comment:
id say the one below is wrong and im thinking about removing it given the download change and its effects.